### PR TITLE
Enable Widget to work in Offline Player

### PR DESCRIPTION
**Pull request by stulees**

### CODE REVIEW

**Comment by rodrigopavezi** [comment]
@stulees @tejohnso please review. cheers

File: bower.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "angular-mocks": "~1.3.0",
     "gsap": "~1.13.2",
     "jquery": "~2.1.1",
-    "rv-common-style": "v1.4.0"
+    "rv-common-style": "v1.4.0",
+    "rise-logger": "1.0.4"
```
Comment (stulees) [inline bower.json]: @rodrigopavezi Not sure why a resolution had to be done for `rise-logger`, can you clarify?

**Comment by rodrigopavezi** [comment]
@stulees I am not sure what happened. It was added the first time I ran bower install. But running it again it is not needed. Please review. cheers

**Comment by stulees** [comment]
@rodrigopavezi 👍

**Comment by rodrigopavezi** [comment]
@tejohnso please review last commit. cheers

**Comment by tejohnso** [comment]
:+1:

**Pull request merged**